pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -1162,22 +1162,29 @@ private void internalGetSubscriptionsForNonPartitionedTopic(AsyncResponse asyncR
                 });
     }
 
-    protected TopicStats internalGetStats(boolean authoritative, boolean getPreciseBacklog,
-                                          boolean subscriptionBacklogSize, boolean getEarliestTimeInBacklog) {
+    protected void internalGetStatsAsync(AsyncResponse asyncResponse, boolean authoritative, boolean getPreciseBacklog,
```

Comment:
Same as this comment https://github.com/apache/pulsar/pull/15347#discussion_r860391215

Reply:
@codelipenghui Fixed.